CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 2.6.2
+
+* [allure] Fixed undefined output directory for allure plugin on using custom runner. Fix by @charliepradeep
```

Comment:
```suggestion
* [WebDriver][Puppeteer] Added `forceClick` method to emulate click event instead of using native events.
* [Playwright] Updated to 0.14
* [Playwright] Introduced `handleDownloads` method to download file. Please note, this method has slightly different API than the same one in Puppeteer.
* [allure] Fixed undefined output directory for allure plugin on using custom runner. Fix by @charliepradeep
```

Reply:
Added my PR changes